business_requirement_renew/models/business_requirement_renew.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+# © 2017 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from odoo import models, api, fields, _
+
+
+class BusinessRequirementRenew(models.Model):
+    _inherit = 'business.requirement'
+
+    source_id = fields.Many2one(
+        'business.requirement',
+        string="Original Bus.Req",
+        readonly=True)
+    copy_from_id = fields.Many2one(
+        'business.requirement', string="Renewed From")
+    version = fields.Integer(string='Version', default=0)
+
+    @api.multi
+    def _compute_br_children_count(self):
+        for record in self:
+            count = len(record.env['business.requirement'].search(
+                [('source_id', '=', record.id)]))
+            record.br_children_count = count + 1
```

Comment:
Should not include the BR itself.
If not renewed it shows "1" (itself), it should show "0"

Reply:
The same behavior should happen when clicking the smart button: the current BR should not be displayed in the list of related BR.